Test/logger/ProofDependencyLogging.dfy

```diff
@@ -2,170 +2,170 @@
-// CHECK:       ProofDependencyLogging.dfy\(219,10\)-\(219,14\): ensures clause
-// CHECK:       ProofDependencyLogging.dfy\(221,2\)-\(221,6\): function call result
-// CHECK:       ProofDependencyLogging.dfy\(221,4\)-\(221,4\): value always satisfies the subset constraints of 'nat'
+// CHECK:       ProofDependencyLogging.dfy\(216,1\)-\(222,1\): function definition for ObviouslyRedundantRequiresFunc
+// CHECK:       ProofDependencyLogging.dfy\(217,12\)-\(217,16\): requires clause
+// CHECK:       ProofDependencyLogging.dfy\(219,11\)-\(219,15\): ensures clause
+// CHECK:       ProofDependencyLogging.dfy\(221,3\)-\(221,7\): function call result
+// CHECK:       ProofDependencyLogging.dfy\(221,5\)-\(221,5\): value always satisfies the subset constraints of 'nat'
 //
 // CHECK: Results for ObviouslyRedundantRequiresMethod \(correctness\)
 // CHECK:     Proof dependencies:
-// CHECK:       ProofDependencyLogging.dfy\(225,11\)-\(225,15\): requires clause
-// CHECK:       ProofDependencyLogging.dfy\(227,10\)-\(227,14\): ensures clause
-// CHECK:       ProofDependencyLogging.dfy\(229,11\)-\(229,11\): value always satisfies the subset constraints of 'nat'
-// CHECK:       ProofDependencyLogging.dfy\(229,2\)-\(229,14\): assignment \(or return\)
+// CHECK:       ProofDependencyLogging.dfy\(225,12\)-\(225,16\): requires clause
+// CHECK:       ProofDependencyLogging.dfy\(227,11\)-\(227,15\): ensures clause
+// CHECK:       ProofDependencyLogging.dfy\(229,12\)-\(229,12\): value always satisfies the subset constraints of 'nat'
+// CHECK:       ProofDependencyLogging.dfy\(229,3\)-\(229,15\): assignment \(or return\)
 //
 // CHECK: Results for ObviouslyUnnecessaryRequiresFunc \(well-formedness\)
 // CHECK:     Proof dependencies:
-// CHECK:       ProofDependencyLogging.dfy\(237,20\)-\(237,20\): value always satisfies the subset constraints of 'nat'
-// CHECK:       ProofDependencyLogging.dfy\(237,31\)-\(237,31\): value always satisfies the subset constraints of 'nat'
+// CHECK:       ProofDependencyLogging.dfy\(237,21\)-\(237,21\): value always satisfies the subset constraints of 'nat'
+// CHECK:       ProofDependencyLogging.dfy\(237,32\)-\(237,32\): value always satisfies the subset constraints of 'nat'
 //
 // CHECK: Results for ObviouslyUnnecessaryRequiresMethod \(correctness\)
 // CHECK:     Proof dependencies:
-// CHECK:       ProofDependencyLogging.dfy\(244,24\)-\(244,24\): value always satisfies the subset constraints of 'nat'
-// CHECK:       ProofDependencyLogging.dfy\(244,47\)-\(244,47\): value always satisfies the subset constraints of 'nat'
+// CHECK:       ProofDependencyLogging.dfy\(244,25\)-\(244,25\): value always satisfies the subset constraints of 'nat'
+// CHECK:       ProofDependencyLogging.dfy\(244,48\)-\(244,48\): value always satisfies the subset constraints of 'nat'
 //
 // CHECK: Results for ObviouslyUnconstrainedCodeFunc \(well-formedness\)
 // CHECK:     Proof dependencies:
```

Comment:
Also out of scope for this review, but can't help ask if we should label this more precisely to account for the fact that some proof dependencies may be "spurious", just as counter examples are really something like "potential counter examples"